src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -110,7 +110,11 @@ public function viewAction(Request $request, $cartId)
     public function getInfoAction(int $cartId)
     {
         try {
-            $cartInfo = $this->getQueryBus()->handle(new GetCartInformation($cartId));
+            $cartInfo = $this->getQueryBus()->handle(
+                (new GetCartForOrderCreation($cartId))
+                    ->setSeparateGiftProducts(false)
+                    ->setSeparateGiftCartRules(false)
```

Comment:
Don't you mean to use `true` here? Otherwise you don't need to call the setters with default values.

Reply:
Yes. I was testing the front in case the values are FALSE and forgot to revert it